src/sentry/integrations/github/client.py

```diff
@@ -114,6 +114,19 @@ def check_file(self, repo, path, version):
         repo_name = repo.name
         return self.head_cached(path=f"/repos/{repo_name}/contents/{path}", params={"ref": version})
 
+    def search_file(self, repo, filename):
+        query = f"filename:{filename}+repo:{repo}"
+        results = self.get(path="/search/code", params={"q": query})
+        return results
+
+    def get_file(self, repo, path):
+        from base64 import b64decode
+
+        # default ref will be the default_branch
```

Comment:
@MeredithAnya where does the ref get set for this API request?

Reply:
github automatically uses the default branch they have set on gh when making this api request if you don't pass a ref. I'm assuming thats what we want since we'd want the latest version of their codeowners